doc/dox/config/Doxyfile.in

```diff
@@ -832,6 +832,7 @@ WARN_LOGFILE           =
 INPUT                  = @PROJECT_SOURCE_DIR@/src \
                          @PROJECT_SOURCE_DIR@/doc/dox/user \
                          @PROJECT_SOURCE_DIR@/doc/dox/dev \
+                         @PROJECT_SOURCE_DIR@/doc/dox/dev/expressions \
```

Comment:
`Customizing-Expressions.md` is the entry-point for users interested in the expression DSL so the notes you added should be referred to from there at a minimum or integrated with that doc at a maximum